extensions/interactions/customization-args-defs.ts

```diff
@@ -25,6 +25,11 @@ import { IReadableNote } from
   // eslint-disable-next-line max-len
   'extensions/interactions/MusicNotesInput/directives/oppia-interactive-music-notes-input.directive';
 
+
+export interface IAlgebraicExpressionInputCustomizationArgs {
```

Comment:
Was there  no customization args for the existing algebraic expression interaction?